### ISSUE TITLE
Avira free anti virus version 14.0.7.468: NVDA does not read the items in the tree view opened through View submenu > PC Protection Submenu > System Scanner

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4804]
**Reported by bhavyashah on 2015-01-14 13:15**
Steps to reproduce
1 Download, install and open Avira free anti virus version 14.0.7.468.
2 Go to the View submenu > PC Protection submenu > System Scanner.
3 Tab to the tree view, possibly your default focus may be there itself.
Expected Result : NVDA will read the names of the items in the tee view, via standard tree view navigation commands, like up and down arrow or through object navigation or screen review.
Actual Result : NVDA instead of reading the tree view items’ names, only says ‘
Tree view item x of 10 level 0’, where x represents the number of the tree view item.   Object navigation and screen review were unsuccessful as well.
Test Environment : Windows 7 Ultimate 32 bit
Avira free anti virus version 14.0.7.468
NVDA version 2014.4 stable
Note: NVDA is presently able to announce in that particular tree view when your focus goes into the tree view, and the number of the item you are on, while Narrator simply remains silent, totally silent for that tree view. I admit that its possible that Narrator may be able to announce it, but isn't doing so presently due to my inexperience.

**Comment by leonardder** [comment]
@bhavyashah: Could you provide a status update or tell whether this is still relevant for you?

**Comment by bhavyashah** [comment]
@leonardder Unfortunately, I do not use Avira anymore. Hence, I will be unable to provide a status update on this issue. However, if there are a bunch of Avira specific issues on the tracker (most of which would probably have been filed by me), I could probably seek assistance from an existing Avira user in testing these by asking on the NVDA Users e-group.

**Comment by Adriani90** [comment]
The interface of Avira changed significantly during last years. Please open a new issue with steps to reproduce according to the new interface. Thanks.
